cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTableAdmin.java

```diff
@@ -198,6 +210,9 @@ private void addIndices(Connection connection, StructuredTableSchema schema) thr
         statement.execute(indexStatement);
       }
     } catch (SQLException e) {
+      if (e instanceof PSQLException && Objects.equals(e.getSQLState(), "42P01")) {
```

Comment:
In short, PostgreSQL error 42P01 denotes the database query is on an undefined table.